### ISSUE TITLE
Setup nginx + php-fpm instead apache

### ISSUE THREAD

**Issue by maksympetrenko** [niiknow/vestacp#16]
Hi,

Can you add configuration for setup nginx + php-fpm instead apache and exclude apache, please.
Or how I can configure it for myself, and care about next updates?

**Comment by noogen** [comment]
Unfortunately, the default php-fpm clean install for ubuntu is broken: https://github.com/serghey-rodin/vesta/issues/1238

nginx + apache + fcgi is very similar anyway (and more flexible) as php-fpm is an alternative fcgi.   I will see what I can do to implement php-fpm in this container; though, it may take a while.

**Comment by noogen** [comment]
nginx + php-fpm will have to be it's own branch or docker build after the fix to the vesta issue reference above.  Instead, I did the next best thing.  I added php-fpm configuration to this container: https://github.com/niiknow/vestacp/wiki/php-fpm-support

Even though Apache2 is installed, it's not using it; and instead, nginx is configured to go directly to php-fpm.  Follow the wiki page above to config/switch between different version of php-fpm.

I will consider this is complete and close this ticket 7 days from now.
